### ISSUE TITLE
I fail to build in manually second-state/ssvm-evmc

### ISSUE THREAD

**Issue by shiki-tak** [second-state/ssvm-evmc#1]
Hello, 
I tried to manually build ssvm-evmc according to the README.
But it failed with the following errors. Did I do something wrong in my configuration?
I used SSVM with version 0.5.1.

```
[ 90%] Linking CXX executable ssvm-aot
/usr/lib/llvm-9/lib/libLLVMCodeGen.a(MIRCanonicalizerPass.cpp.o): In function `rescheduleLexographically(std::vector<llvm::MachineInstr*, std::allocator<llvm::MachineInstr*> >, llvm::MachineBasicBlock*, std::function<llvm::MachineInstrBundleIterator<llvm::MachineInstr, false> ()>)':
MIRCanonicalizerPass.cpp:(.text._ZL25rescheduleLexographicallySt6vectorIPN4llvm12MachineInstrESaIS2_EEPNS0_17MachineBasicBlockESt8functionIFNS0_26MachineInstrBundleIteratorIS1_Lb0EEEvEE+0x167): undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long, unsigned long)'
/usr/lib/llvm-9/lib/libLLVMInstrumentation.a(GCOVProfiling.cpp.o): In function `llvm::GCOVOptions::getDefault()':
GCOVProfiling.cpp:(.text._ZN4llvm11GCOVOptions10getDefaultEv+0x7b): undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&)'
/usr/lib/llvm-9/lib/libLLVMInstrumentation.a(GCOVProfiling.cpp.o): In function `llvm::Pass* llvm::callDefaultCtor<(anonymous namespace)::GCOVProfilerLegacyPass>()':
GCOVProfiling.cpp:(.text._ZN4llvm15callDefaultCtorIN12_GLOBAL__N_122GCOVProfilerLegacyPassEEEPNS_4PassEv+0x1ee): undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&)'
/usr/lib/llvm-9/lib/libLLVMMC.a(MCContext.cpp.o): In function `llvm::MCContext::RemapDebugPaths()::$_1::operator()(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&) const':
MCContext.cpp:(.text._ZZN4llvm9MCContext15RemapDebugPathsEvENK3$_1clERNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE+0x97): undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long, unsigned long)'
/usr/lib/llvm-9/lib/libLLVMCodeGen.a(GCMetadata.cpp.o): In function `llvm::GCModuleInfo::getGCStrategy(llvm::StringRef)':
GCMetadata.cpp:(.text._ZN4llvm12GCModuleInfo13getGCStrategyENS_9StringRefE+0x2f1): undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&)'
/usr/lib/llvm-9/lib/libLLVMCodeGen.a(RegisterScavenging.cpp.o): In function `llvm::RegScavenger::spill(unsigned int, llvm::TargetRegisterClass const&, int, llvm::MachineInstrBundleIterator<llvm::MachineInstr, false>, llvm::MachineInstrBundleIterator<llvm::MachineInstr, false>&)':
RegisterScavenging.cpp:(.text._ZN4llvm12RegScavenger5spillEjRKNS_19TargetRegisterClassEiNS_26MachineInstrBundleIteratorINS_12MachineInstrELb0EEERS6_+0x346): undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&)'
/usr/lib/llvm-9/lib/libLLVMCore.a(Mangler.cpp.o): In function `llvm::emitLinkerFlagsForGlobalCOFF(llvm::raw_ostream&, llvm::GlobalValue const*, llvm::Triple const&, llvm::Mangler&)':
Mangler.cpp:(.text._ZN4llvm28emitLinkerFlagsForGlobalCOFFERNS_11raw_ostreamEPKNS_11GlobalValueERKNS_6TripleERNS_7ManglerE+0x179): undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long, unsigned long)'
/usr/lib/llvm-9/lib/libLLVMSupport.a(ErrorHandling.cpp.o): In function `llvm::report_bad_alloc_error(char const*, bool)':
ErrorHandling.cpp:(.text._ZN4llvm22report_bad_alloc_errorEPKcb+0xa4): undefined reference to `std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(char const*, std::allocator<char> const&)'
collect2: error: ld returned 1 exit status
utils/ssvm-core/tools/ssvm-aot/CMakeFiles/ssvm-aot.dir/build.make:154: recipe for target 'utils/ssvm-core/tools/ssvm-aot/ssvm-aot' failed
make[2]: *** [utils/ssvm-core/tools/ssvm-aot/ssvm-aot] Error 1
CMakeFiles/Makefile2:2561: recipe for target 'utils/ssvm-core/tools/ssvm-aot/CMakeFiles/ssvm-aot.dir/all' failed
make[1]: *** [utils/ssvm-core/tools/ssvm-aot/CMakeFiles/ssvm-aot.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by q82419** [comment]
Hello,
We suggest to build ssvm-evmc with master branch of SSVM for now.
We'll release 0.6.0 soon to fix some issues.
To use master branch of SSVM, you can use default option with not assigning `SSVM_CORE_PATH` flag in cmake.
Thank you.

**Comment by hydai** [comment]
Hi @shiki-tak 
We've released [SSVM 0.6.0](https://github.com/second-state/SSVM/releases/tag/0.6.0) just now. Please use this version for building SSVM-EVMC.

Thanks.

**Comment by shiki-tak** [comment]
> Hi @shiki-tak
> We've released [SSVM 0.6.0](https://github.com/second-state/SSVM/releases/tag/0.6.0) just now. Please use this version for building SSVM-EVMC.
> 
> Thanks.

Thanks, I'll try it.
